packages/web/src/components/data-entry/TagInput.d.ts

```diff
@@ -0,0 +1,16 @@
+export type TagInputProps = {
```

Comment:
Nice. Someday we can move this component to ts, but happy to have this for now

Reply:
yes exactly, had to do that for a few of these doods